src/_posts/whats-new-in-0.16.0.md

```diff
@@ -0,0 +1,127 @@
+
+List of new packages:
+
+* [`infection/codeception-adapter`](https://github.com/infection/codeception-adapter)
+* [`infection/phpspec-adapter`](https://github.com/infection/phpspec-adapter)
+* [`infection/abstract-testframework-adapter`](https://github.com/infection/abstract-testframework-adapter) - interfaces for any testframework adapter
+* [`infection/extension-installer`](https://github.com/infection/extension-installer) - autodiscovery of installed adapters. No need to add them to `infection.json`, just install a Composer package and you are done. Inspired by PHPStan and Psalm plugin system.
+* [`infection/include-interceptor`](https://github.com/infection/include-interceptor) - Stream Wrapper. Allows to replace included (autoloaded) file with another one (magic that replaces your code with a mutated one) 
+
+
+## Mutation Badge
+
+We had this feature [before](/guide/mutation-badge.html), but it worked only for TravisCI.
+
+Now, thanks to [`ondram/ci-detector`](https://github.com/OndraM/ci-detector), the mutation badge can be used for [CI servers supported by this library](https://github.com/OndraM/ci-detector#supported-continuous-integration-servers), including GitHub Actions, CircleCI, GitLab and so on.
+
+Feel free to request an additional integration there if the list of currently supported continuous-integration servers does not contain a service you require.
+
+> Read more about how to setup [Mutation Badge](/guide/mutation-badge.html)
+
+## Global ignore feature for Mutators and Profiles
+
+`global-ignore` allows to apply the `ignore` setting to all mutators and profiles registered. If `ignore` is specified for a given profile or mutator, it will be merged with `global-ignore`. 
+
+For example, in the case below 
+
+
+```json
+{
+    "mutators": {
+        "@default": true,
+        "global-ignore": [
+            "FooClass::__construct"
+        ],
+        "TrueValue": {
+            "ignore": [
+                "NameSpace\\*\\SourceClass::create",
+                "Full\\NameSpaced\\Class"
+            ]
+        }
+    }
+}
+```
+
+the final `ignore` setting for `TrueValue` will be:
+
+```json
+[
+    "FooClass::__construct",
+    "NameSpace\\*\\SourceClass::create",
+    "Full\\NameSpaced\\Class"
+]
+```
+
+## Notice to increase min MSI metrics
+
+It's possible to [use Infection with CI server](/guide/using-with-ci.html), adding `--min-msi` and/or `--min-covered-msi` options, so when the actual results of these metrics are below the threshold, the build fails.
+
+But, when you constantly improve the quality of your tests, MSI metric eventually increases its value (which is good), and now Infection will remind you to update the limits.
+
+```bash
+! [NOTE] The MSI is 2.5787422095418% percent points over the required MSI.     
+!        Consider increasing the required MSI percentage the next time you run 
+!        infection.    
+```
```

Comment:
```suggestion
While this is not the recommended way*, it is possible to use the options `--min-msi` and/or `--min-covered-msi` options to force a certain threshold (the Infection process will exit with the error code `1` if one of those is not reached). When using Infection with those, Infection would also now display a recommendation to increase this threshold when the score is above the configured threshold:

```bash
! [NOTE] The MSI is 2.5787422095418% percent points over the required MSI.     
!        Consider increasing the required MSI percentage the next time you run 
!        infection.    
```

*: The MSI score is but a metric like another. The interesting part is not necessarily the score but which mutations escaped and whether or not those should be caught. Moreover if you are using Infection in an incremental way, i.e. running it only the changed files, this metric will drastically vary from a change to another depending of how testing this part is making it unsuitable for this use case.
```